gensrc/thrift/FrontendService.thrift

```diff
@@ -654,6 +654,25 @@ struct TSnapshotLoaderReportRequest {
     5: optional i32 total_num
 }
 
+enum TFrontendPingFrontendStatusCode {
+   OK = 0,
+   FAILED = 1
+}
+
+struct TFrontendPingFrontendRequest {
+   1: required i32 clusterId
+   2: required string token
+}
+
+struct TBootstrapResult {
+    1: required TFrontendPingFrontendStatusCode status
```

Comment:
Is it ok to use `TStatus` here?

Reply:
TStatus is used for be, just like TBrokerOperationStatus is used for broker, so I think use new status for fe would be better

Reply:
Didn't find any specification that `TStatus` can only be used between BE and FE.
https://github.com/apache/incubator-doris/blob/49f7eb69bfa9190612e84d5cbe989fe15fb4e995/gensrc/thrift/Status.thrift#L22

Reply:
Yes, I am also not sure to reuse the TStatus or just like 
```

enum TBrokerOperationStatusCode {
    OK = 0;
    END_OF_FILE = 301;
    
    // user input error
    NOT_AUTHORIZED = 401;
    DUPLICATE_REQUEST = 402;
    INVALID_INPUT_OFFSET = 403; // user input offset is invalid, is large than file length
    INVALID_INPUT_FILE_PATH = 404;
    INVALID_ARGUMENT = 405;
    
    // internal server error
    FILE_NOT_FOUND = 501;
    TARGET_STORAGE_SERVICE_ERROR = 502; // the target storage service error
    OPERATION_NOT_SUPPORTED = 503; // the api is not implemented
}
```
define new status struct